lib/components/serverside-data/Table.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <v-data-table
+    class="gever-table"
+    dense
+    :disable-pagination="disablePagination"
+    :hide-default-footer="disablePagination"
+    :options="options"
+    v-bind="{ ...$attrs, headers: tableHeaders }"
+  >
+    <template #body="{ items }">
+      <tbody v-if="$attrs.loading">
+        <tr>
+          <td colspan="100">
+            <v-skeleton-loader type="list-item-two-line@4" />
+          </td>
+        </tr>
+      </tbody>
+      <tbody v-else-if="!$attrs.loading && items.length === 0">
+        <tr>
+          <td colspan="100" class="text-center empty-cell py-5">
+            {{ $t('table.emptyState') }}
+          </td>
+        </tr>
+      </tbody>
+      <template v-else>
+        <tbody v-for="item in items" :id="item.id" :key="item.id" :class="{ selected: $route.hash === `#${item.id}` }">
+          <tr>
+            <td v-if="$attrs.icon" class="icon-cell pl-6 pr-1" rowspan="2" style="width: 0">
+              <v-icon v-if="$attrs.icon.startsWith('mdi')" width="18px" height="18px">{{ $attrs.icon }}</v-icon>
+              <svg-icon v-else :name="$attrs.icon" width="18px" height="18px" />
```

Comment:
```suggestion
              <v-icon width="18px" height="18px">{{ $attrs.icon }}</v-icon>
```
Ich würde hier nur mal mdi icons anbieten. Wie gesagt schauen wir svg-icons später an.

Reply:
👍🏼  done.